testdata/dnn/download_models.py

```diff
@@ -1000,6 +1000,16 @@ def get_confirm_token(response):  # in case of large files
         downloader=GDrive('1rm3mIqjJNu0xPTCjMKnXccspazV1B2zv'),
         sha='fcd0fa401c83bf2b29e18239a9c2c989c9b8669d',
         filename='onnx/models/yolov7_not_simplified.onnx'),
+    Model(
+        name='NanoTrack (ONNX)',
+        url='https://github.com/zihaomu/opencv_extra_data_backup/raw/main/NanoTrack/models/nanotrack_backbone_sim.onnx',
```

Comment:
> /main/

Please use exact commit SHA instead of common branch name

---

> github.com

Also we could try to use `raw.githubusercontent.com` instead.

Reply:
Hi, @alalek. Thx for your code reviewing, I Fixed.